src/block_service/fds/fds_service.cpp

```diff
@@ -533,11 +533,14 @@ error_code fds_file_object::put_content(/*in-out*/ std::istream &is,
     galaxy::fds::GalaxyFDSClient *c = _service->get_client();
 
     // get tokens from token bucket
-    if (!_service->_write_token_bucket->consumeWithBorrowAndWait(to_transfer_bytes)) {
-        ddebug_f("the transfer count({}) is greater than burst size({}), so it is rejected by "
+    if (!_service->_write_token_bucket->consumeWithBorrowAndWait(to_transfer_bytes,
+                                                                 FLAGS_fds_write_limit_rate << 20,
+                                                                 FLAGS_fds_write_burst_size
+                                                                     << 20)) {
+        ddebug_f("the transfer count({}) is greater than burst size({}MB), so it is rejected by "
```

Comment:
`the transfer count({})` byte maybe better? Because the burst size has unit(MB)

Reply:
good idea :）